model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -68,6 +68,24 @@ data class OrtConfiguration(
      */
     val enableRepositoryPackageConfigurations: Boolean = false,
 
+    /**
+     * A set with the names of environment variables that are passed to child processes created by ORT, e.g. when
+     * running external package managers or scanners. All variables not listed here are not passed through. This
+     * property aims to prevent that sensitive information from environment variables like passwords or service
+     * credentials is passed to external tools. As a default value, a larger number of well-known variables evaluated
+     * by the tools invoked by ORT is set. Users can choose to override this by redefining this property in their
+     * configuration file. In order to extend the set, the [additionalVariablesToPropagate] property should be used
+     * instead.
+     */
+    val variablesToPropagate: Set<String> = emptySet(),
```

Comment:
I'd prefer if the variable name would contain the word "environment". Maybe something like `allowedProcessEnvironmentVariables`?

Reply:
Changed implementation.